src/EmployerAan/SFA.DAS.EmployerAan/InnerApi/Notifications/Requests/PostNotificationRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerAan.Infrastructure;
+using SFA.DAS.EmployerAan.InnerApi.Notifications.Responses;
+
+namespace SFA.DAS.EmployerAan.Application.InnerApi.Notifications;
+
+public record PostNotificationRequest() : IRequest<GetNotificationResponse>
+{
+    [FromHeader(Name = Constants.ApiHeaders.RequestedByMemberIdHeader)]
+    public Guid RequestedByMemberId { get; set; }
+    [FromQuery]
+    public Guid MemberId { get; set; }
+    [FromQuery]
+    public int NotificationTemplateId { get; set; }
```

Comment:
This attribute decoration is wrong. Remove all the decorations. Also remove the RequestedByMemberId  property as that is injected explicitly

Reply:
Done